### DISCUSSION TITLE
All imported Elements have transparent BG's?

### DISCUSSION THREAD

**Discussion by HobbyHaus** [mdbootstrap/TW-Elements#1583] [Support from community]
Hello all,

No matter the element, when I use it, the drop down or date picker (or whatever the element) has a seemingly transparent background.
---------------
What I've tried

1. Disabling my CSS
2. adjusting z-index
3. targeting the background of elements (which is seemingly impossible)

Nothing works. Have had nothing but issues using this easy setup. Hoping someone here can help shed some light. 

---------------
link to screenshot of one (date picker) example. The text "tab 1 content vertical" is behind the pop up date picker window
https://i.ibb.co/9VxDxC8/Screenshot-3.jpg

**Comment by smolenski-mikolaj** [comment]
@HobbyHaus did you set `darkMode: "class"` in tailwind config file?

**Reply by HobbyHaus** [reply]
Yes, I just checked and thats in there.

**Reply by smolenski-mikolaj** [reply]
I recommend to check our [MDB GO Starter](https://tailwind-elements.com/docs/standard/getting-started/quick-start/#go) to compare the configuration of the projects.

**Reply by HobbyHaus** [reply]
Okay the only "fix" (which isn't a fix) comes when I place class="dark" within the browser inspector, seen below...
![image](https://user-images.githubusercontent.com/7841709/231560607-e9dfd292-c542-4951-af6a-a7a348d07d6e.png)
that alternation, on a browser level, nets me this....
![image](https://user-images.githubusercontent.com/7841709/231560750-919f0d7f-bd2d-4a4c-b4a2-23f94576e4cf.png)
important thing to note, nothing "fixes" if I substitute light instead of dark.

**Reply by smolenski-mikolaj** [reply]
Do you have tailwind modules imported like this in your main css/scss file?

```
@tailwind base;

@layer base {
  html {
    @apply text-neutral-800;
  }
  html.dark {
    @apply text-neutral-50;
    @apply bg-neutral-800;
  }
}

@tailwind components;
@tailwind utilities;
```

**Reply by HobbyHaus** [reply]
Yes, exactly like this.
![image](https://user-images.githubusercontent.com/7841709/232930196-ac050918-7018-4443-b596-a3aa76ddfa31.png)

**Reply by HobbyHaus** [reply]
Just to reiterate, this issue is NOT fixed.
